wolfcrypt/src/ecc.c

```diff
@@ -10671,7 +10671,7 @@ static int wc_ecc_import_raw_private(ecc_key* key, const char* qx,
     }
 
     if (err == MP_OKAY) {
-        if (mp_iszero(key->pubkey.x) && mp_iszero(key->pubkey.y)) {
+        if (mp_iszero(key->pubkey.x) || mp_iszero(key->pubkey.y)) {
```

Comment:
One of the ordinates can be zero and it still be a valid point.
y^2 = x^3 + ax + b, when x = 0, y = sort(b)

Wycheproof tests fail when you change this to ||.
(ecdh_webcrypto_test: test numbers 30, 58, 237, 294, 418, 513)

Reply:
Reverted